src/UI/README.md

```diff
@@ -391,6 +391,118 @@ If you would like to implement a new component to the framework you should perfo
 
 TODO: write me
 
+## Abstraction of Javascript in the Framework
```

Comment:
This sounds like a very promising start for the much needed documentation/concept about this.